src/client/components/LogView.tsx

```diff
@@ -63,12 +68,21 @@ class LogView extends Component<LogViewProps, LogViewState> {
     })
   }
 
+  findShiftedTimezoneDate = (logTimestamp, timezoneName): string => {
+    const logDateUtc = new Date(logTimestamp * 1000)
+    return DateTime.fromISO(logDateUtc.toJSON(), { zone: timezoneName }).toISO()
+  }
+
```

Comment:
No need to use a function here, it's only used once, it's super simple, and you are actually just making the code longer and more convoluted this way.

Reply:
Updated per comment.